### PR TITLE
[POC] new Ricos viewer

### PR DESCRIPTION
**Pull request by oded-soffrin**
image

### CODE REVIEW

**Comment by lxgreen** [comment]
image

**Comment by lxgreen** [comment]
image

File: packages/ricos-common/web/src/RicosEngine.tsx [hunk 1]
```diff
@@ -41,7 +43,7 @@ export class RicosEngine extends Component<EngineProps> {
 
     const strategiesProps = merge(
       { theme },
-      pluginsStrategy(isViewer, plugins, children.props, theme, content)
+      pluginsStrategy(isViewer, plugins, children.props, content)
```
Comment (Barackos) [inline packages/ricos-common/web/src/RicosEngine.tsx:46]: @justame pluginsStrategy requires this `theme` (a.k.a `cssOverride`)

**Comment by oded-soffrin** [comment]
@danarish4 this is the latest efforts of non-draft based viewer, before we hold it to improve current viewer performance and focus on the editor. keep in mind for when we'll start working on the new viewer again

**Pull request closed without merging**